app/workers/reactions/bust_homepage_cache_worker.rb

```diff
@@ -0,0 +1,22 @@
+module Reactions
+  class BustHomepageCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(reaction_id)
+      cache_buster = CacheBuster
+      reaction = Reaction.find_by(id: reaction_id, reactable_type: "Article")
+      return unless reaction&.reactable
+
+      featured_articles_ids = Article.where(featured: true).order("hotness_score DESC").limit(3).pluck(:id)
```

Comment:
It looks like these 3 article IDs don't change that often. Might be worth adding a short cache here at some point

Reply:
With  `CacheBuster`?

Reply:
@mstruve maybe you can test with `psql` if `EXPLAIN ANALYZE QUERY` is faster than the cache roundtrip. 

The final query should be:

```sql
SELECT  id FROM "articles" WHERE "articles"."featured" = TRUE ORDER BY hotness_score DESC LIMIT 3
```